build/xtask/src/dist.rs

```diff
@@ -2575,6 +2626,8 @@ fn objcopy_translate_format(
         .arg(out_format)
         .arg("--gap-fill")
         .arg("0xFF")
+        .arg("--srec-forceS3") // Manually constructed Srecords use the S3 format
+        .arg("--srec-len=255") // Objcopy will select a shorter line length if allowed, this forces it to match the manual Srecord construction.
```

Comment:
Do these arguments need to be conditionally added if the out_format is srec?

Reply:
No - objcopy ignores them entirely for non-srec outputs.